tests/CMakeLists.txt

```diff
@@ -0,0 +1,29 @@
+#
+# Copyright (C) 2024 Renesas Electronics Corporation.
+# Copyright (C) 2024 EPAM Systems, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+set(TARGET aos_iamanager_test)
+
+# ######################################################################################################################
+# Sources
+# ######################################################################################################################
+
+set(SOURCES ../../src/logger/logger.cpp src/exception_test.cpp)
```

Comment:
I would better make test/CMakeLists.txt to include cmake files as sub_directories.